src/com/form.js

```diff
@@ -254,8 +249,8 @@
 					},
 					// must return a string
 					format: function (value, options) {
-						if (!moment.isMoment(value)) {
-							value = moment(value);
+						if (!win.moment.isMoment(value)) {
+							value = win.moment(value);
```

Comment:
```suggestion
							value = global.moment(value);
```

Reply:
@nitriques you are commenting on outdated code. This is already resolved. see 7e5b8fc